rhino-core/src/main/java/io/ryos/rhino/sdk/SimulationJobsScannerImpl.java

```diff
@@ -226,11 +226,11 @@ private SimulationMetadata createBenchmarkJob(final Class clazz) {
   }
 
   private boolean isBlockingSimulation(final Runner runnerAnnotation) {
-    return runnerAnnotation.clazz().equals(DefaultSimulationRunner.class);
```

Comment:
If `@Runner` is missing, then isBlockingRunner = true so we use DefaultSimulationRunner in absence.